brycemara/taco-bout-it
# Feature/aboutus

## Is this a fix or a feature?
Feature.

## What is the change?
Creates the About component. This just renders static information about the taco truck.
Also I have changed the FindUs section to be a Reviews tab. Find Us is now located in the About.

## Where should the reviewer start?
`About.js`
`About.test.js`


## How should this be tested?
run `npm test`
Watch tests pass.
Needs more testing to check for correct rendering of the Come Visit Us section. 

## What are the relevant tickets?
Ticket #10 & #6 

## Screenshots (if appropriate)

![Screen Shot 2021-01-13 at 1 32 53 PM](https://user-images.githubusercontent.com/68242949/104500296-dc5c5180-55a3-11eb-8fe3-78e1d0b829d5.png)


@brycemara
